src/posix.mak

```diff
@@ -211,6 +200,11 @@ ifeq ($(CXX_KIND), g++)
 CXXFLAGS += \
     -std=gnu++98
 endif
+# Clang Specific
+ifeq ($(CXX_KIND), clang++)
```

Comment:
FYI `g++` supports this flag too (and actually my local `clang++` doesn't trigger a warning without it). In any case I don't think there's any harm in always letting `-xc++` be part of the `CXXFLAGS`.

Reply:
I must have been doing something weird the first time I ran make with cxx=clang because I couldn't reproduce after the a couple runs, maybe I added or removed something before resetting the branch, such as deprecation warnings.